src/docs/development/platform-integration/c-interop.md

```diff
@@ -351,6 +351,35 @@ in binary form, use the following instructions:
 
 This feature is not yet supported for web plugins.
 
+## FAQ
+
+### Android APK size (shared object compression)
+
+Android guidelines in general recommend distributing native shared objects uncompressed because that actually saves on device space. Shared objects can be directly loaded from the APK instead of unpacking them on device into a temporary location and then loading. APKs are additionally packed in transit - that is why you should be looking at download size. 
```

Comment:
I think you should link to https://developer.android.com/topic/performance/reduce-apk-size#extract-false in this sentence somewhere.